### DISCUSSION TITLE
Error when trying to use Pull Request Manager Hub

### DISCUSSION THREAD

**Discussion by avidor-optimove** [cribeiro84/azure-devops-pull-request-hub#194] [General]
Hello,

I installed this Addon in my Azure Devops project but when trying to run this addon I get the following error:
There was an error during the extension load: TFS.WebApi.Exception: TF401027: You need the Git 'GenericRead' permission to perform this action.

I have the full permissions for my account but anyway it still fails to load the Pull Requests
Also when trying to read more about this error over the web, it looks like this error refers to something else: Git 'CreateRepository' permission

I would appriciate any help
Thanks in advance
Avidor

**Comment by cribeiro84** [comment]
Hello there
That's strange. It was happening when there is any disabled repo in your list, however, it was supposed to be fixed. Could you please check if you have any disabled repo in your project?

Thanks

**Reply by avidor-optimove** [reply]
Seems like we do have some disabled repos, but I can't enable them, I asked the system admin here to check for this issue and I will update if it solved the problem

**Comment by cribeiro84** [comment]
Got it, it was supposed to be fixed already, that's strange. It's now working for the other issue open before similar to this one.
